### PR TITLE
i18n(pt-BR): Create `authentication.mdx`

### PR DESCRIPTION
**Pull request by thekidisalright**
#### Description (required)
authentication.mdx file translated to pt-BR.

The file did not exist in the pt-BR documentation, so I created and translated from the original one.

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by afucher** [comment]
Hi @thekidisalright ! 
Can you update the PR title following [this guidelines](https://contribute.docs.astro.build/guides/i18n/#pull-request-guidelines) please?
It helps when filtering open PRs, and keep consistency

File: src/content/docs/pt-br/guides/authentication.mdx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  <Fragment slot="yarn">
+  ```shell
+  yarn add lucia
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Configuração
+
+Use [o guia "Getting started in Astro"](https://lucia-auth.com/getting-started/astro) para inicializar a Lucia com um adaptador e implementar um banco de dados para armazenar usuários e sessões.
```
Comment (afucher) [inline src/content/docs/pt-br/guides/authentication.mdx:48]: We translate titles also, if you want you can add a note to warn the user that the content is in English. We have some other examples, I think that in the community recipes.
```suggestion
Use [o guia "Começando com Astro" da Lucia](https://lucia-auth.com/getting-started/astro) para inicializar a Lucia com um adaptador e implementar um banco de dados para armazenar usuários e sessões.
```

File: src/content/docs/pt-br/guides/authentication.mdx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+:::
+
+### Próximos passos
+
+- [Examplo Astro + projeto Lucia OAuth](https://github.com/lucia-auth/examples/tree/main/astro/github-oauth)
+- [Examplo Astro + projeto Lucia com nome de usuário e senha](https://github.com/lucia-auth/examples/tree/main/astro/username-and-password)
+
+## Auth.js
+
+Auth.js é uma solução para autenticação que independe de frameworks. Um adaptador de framework [`auth-astro`](https://www.npmjs.com/package/auth-astro) está disponível para o Astro.
```
Comment (afucher) [inline src/content/docs/pt-br/guides/authentication.mdx:63]:
```suggestion
Auth.js é uma solução para autenticação que independe de frameworks. Um adaptador do framework, feito pela comunidade, [`auth-astro`](https://www.npmjs.com/package/auth-astro) está disponível para o Astro.
```

File: src/content/docs/pt-br/guides/authentication.mdx [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+### Instalação
+
+Use o comando `astro add` para que seu gerenciador de pacotes preferido adicione a integração `astro-auth`.
```
Comment (afucher) [inline src/content/docs/pt-br/guides/authentication.mdx:67]:
```suggestion
Use o comando `astro add` do seu gerenciador de pacotes preferido para adicionar a integração `astro-auth`.
```

**Review by afucher — COMMENTED** [review]
Nice contribution! Added a few comments 😇

**Review by afucher — APPROVED** [review]
LGTM 🎉

File: src/content/docs/pt-br/guides/authentication.mdx [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+  yarn add lucia
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Configuração
+
+Use [o guia "Começando com Astro" da Lucia](https://lucia-auth.com/getting-started/astro) para inicializar a Lucia com um adaptador e implementar um banco de dados para armazenar usuários e sessões.
+
+### Usage
```
Comment (yanthomasdev) [inline src/content/docs/pt-br/guides/authentication.mdx:50]:
```suggestion
### Utilização
```

File: src/content/docs/pt-br/guides/authentication.mdx [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+Autenticação e autorização são dois processos de segurança que te ajudam a controlar quem possui acesso a diferentes parte do seu website ou aplicativo. Autenticação é o processo de verificar a identidade de um visitante, e autorização é o processo de permitir aos visitantes acesso a recursos e rotas protegidas.
+
+Autenticação te permite customizar áreas do seu site para indivíduos logados e fornece proteção máxima a informações pessoais ou privadas. Bibliotecas de autenticação  (e.x. [Lucia Auth](https://lucia-auth.com/), [Auth.js](https://authjs.dev/)) fornecem utilidades para múltiplos métodos de autenticação como acesso por email e provedores OAuth.
+
+:::tip
+Não existe solução oficial para autenticação no Astro, mas você pode encontrar [integrações "auth" da comunidade](https://astro.build/integrations/?search=auth) no diretório de integrações.
+:::
+
+<ReadMore>Veja como [adicionar autenticação com Supabase](/en/guides/backend/supabase/#adding-authentication-with-supabase) ou [adicionar autenticação com Firebase](/en/guides/backend/google-firebase/#adding-authentication-with-firebase) em nossos guias dedicados a esses serviços backend.</ReadMore>
```
Comment (yanthomasdev) [inline src/content/docs/pt-br/guides/authentication.mdx:18]:
```suggestion
<ReadMore>Veja como [adicionar autenticação com Supabase](/pt-br/guides/backend/supabase/#adding-authentication-with-supabase) ou [adicionar autenticação com Firebase](/pt-br/guides/backend/google-firebase/#adding-authentication-with-firebase) em nossos guias dedicados a esses serviços backend.</ReadMore>
```

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work peeps 🙌

**Pull request merged**